modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+
+[id="chains-new-features-1-7_{context}"]
+=== Chains
+
+[IMPORTANT]
+====
+Tekton Chains is a Technology Preview feature.
+====
+
+Tekton Chains is a Kubernetes Custom Resource Definition (CRD) controller. You can use it to manage the supply chain security of the tasks and pipelines created using {pipelines-title}.
+
+By default, Tekton Chains observes all task run executions in your {product-title} cluster. When the task runs complete, Chains takes a snapshot of the task runs. It then converts the snapshot to one or more standard payload formats, and finally signs and stores all artifacts.
+
+Tekton Chains supports the following features:
+
+* You can sign task runs, task run results, and OCI registry images with cryptographic key types and services such as `cosign`.
+
+* You can use attestation formats such as `in-toto`.
+
+* You can securely store signatures and signed artifacts using OCI repository as a storage backend.
+
+[id="pac-new-features-1-7_{context}"]
+=== Pipelines as Code (PAC)
+
+[IMPORTANT]
+====
+Pipelines as Code is a Technology Preview feature.
+====
+
+With Pipelines as Code, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports status.
+
+Pipelines as Code supports the following features:
+
+* Pull request status. When iterating over a pull request, the status and control is executed on the platform hosting the Git repository.
+
+* GitHub Checks API to set the status of a pipeline run, including rechecks.
+
+* GitHub Pull Request and Commit events. 
+
+* Pull request actions in comments, such as `/retest`.
+
+* Git events filtering, and a separate pipeline for each event.
+
+* Automatic task resolution in {pipelines-shortname} (local tasks, Tekton Hub, and remote URLs).
+
+* Use of GitHub blobs and objects API for retrieving configurations.
+
+* Access Control List (ACL) over a GitHub organization, or using a Prow style `OWNER` file.
+
+* The `tkn-pac` plugin for the `tkn` CLI tool, which you can use to manage {pac} repositories and bootstrapping.
+
+* Support for GitHub Application, GitHub Webhook, Bitbucket Server, and Bitbucket Cloud.
+
+[id="deprecated-features-1-7_{context}"]
+== Deprecated features
+
+// Pipelines
+* Breaking change: The `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` custom resource (CR) is removed. As a result, the `$HOME` environment variable and `workingDir` parameter are unavailable.
+// https://github.com/tektoncd/pipeline/pull/4587
+
+* The `Conditions` custom resource definition (CRD) type will be removed in a future release. Instead, use the recommended `When` expression.
```

Comment:
Please use the standard deprecation notice format:
https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Reply:
The `Conditions` custom resource definition (CRD) type is deprecated and planned to be removed in a future release.